### PR TITLE
Styling changes and documented functions

### PR DESCRIPTION
**Pull request by l2zou**
- Separated _add_primitive into _add_dict, _add_list, and _add_set
- Changed variable names to be more descriptive
- Added docstrings for dyndict.py and test.py
- Other minor styling changes

### CODE REVIEW

**Comment by HillcrestEnigma** [comment]
Thank you for your PR. The code looks good and I will review soon. Could you modify the code to make the linter more happier in the meantime?

**Comment by l2zou** [comment]
Linter warnings have been resolved.

File: dyndict/dyndict.py [hunk 1]
```diff
@@ -2,80 +2,113 @@
...
+            else:
+                a[key] = copy.deepcopy(b[key])
+        return a
+
+    def _add_list(self, a: list, b: list):
+        """
+        Adds two lists according to dyndict rules.
+        """
+        sum_list = copy.deepcopy(a) + copy.deepcopy(b)
+        name_dict = dict()
```
Comment (HillcrestEnigma) [inline dyndict/dyndict.py:41]: why `name_dict`?

Comment (HillcrestEnigma) [reply dyndict/dyndict.py:41]: nvm I was trolling

**Review by HillcrestEnigma — APPROVED** [review]

**Comment by HillcrestEnigma** [comment]
thanks!

**Pull request merged**